### PR TITLE
PRIME-2812 fix enrollee stuck on certification page

### PR DESCRIPTION
**Pull request by bergomi02**

### CODE REVIEW

**Comment by neophyte57** [comment]
Instead of indicating which fields are Required, just a generic error? 
image

**Comment by bergomi02** [comment]
> Instead of indicating which fields are Required, just a generic error? image

This is fixed. Also fixed the issue that practitioner ID field remains visible if previously select nurse license class.

File: prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts [hunk 1]
```diff
@@ -25,6 +25,36 @@ export enum CollegeLicenceGroupingEnum {
...
+  CollegeLicenceGroupingEnum.HealthProfessionOfAudiology,
+  CollegeLicenceGroupingEnum.HealthProfessionOfHearingInstrumentDispensing,
+  CollegeLicenceGroupingEnum.HealthProfessionOfSpeechLanguagePathology,
+  CollegeLicenceGroupingEnum.HealthProfessionOfChiropractic,
+  CollegeLicenceGroupingEnum.HealthProfessionOfMassageTherapy,
+  CollegeLicenceGroupingEnum.HealthProfessionOfNaturopathicMedicine,
+  CollegeLicenceGroupingEnum.HealthProfessionOfTraditionalChineseMedicineAcupuncture
+]
+
 export const LicenseGrouping: number[] = [
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts:58]: Can `LicenseGrouping` be a union of `NursingLicenseGrouping` and `NonNursingLicenseGrouping` instead?

Comment (bergomi02) [reply prime-angular-frontend/src/app/shared/enums/college-licence-grouping.enum.ts:58]: Sure. Code is updated.

File: prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ import { CollegeLicenceClassEnum } from '@shared/enums/college-licence-class.enu
 import { PrescriberIdTypeEnum } from '@shared/enums/prescriber-id-type.enum';
 import { CollegeCertification } from '@enrolment/shared/models/college-certification.model';
 import { EnrolmentService } from '@enrolment/shared/services/enrolment.service';
-import { LicenseGrouping } from '@shared/enums/college-licence-grouping.enum';
+import { LicenseGrouping, NonNursingLicenseGrouping, NursingLicenseGrouping } from '@shared/enums/college-licence-grouping.enum';
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts:19]: So `LicenseGrouping` import and definition can be removed now, correct?

File: prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts [hunk 3]
```diff
@@ -351,6 +355,15 @@ export class CollegeCertificationFormComponent implements OnInit {
     }
   }
 
+  private setGroupingLicenseValidators(): void {
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts:358]: Better renamed to `setNonNursingLicenseGroupingValidators`?

**Review by neophyte57 — APPROVED** [review]
Great job diagnosing and fixing!

**Pull request merged**